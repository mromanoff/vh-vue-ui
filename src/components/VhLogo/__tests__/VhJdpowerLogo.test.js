import { shallowMount } from '@vue/test-utils';
import VhJdpowerLogo from '../VhJdpowerLogo.vue';

describe('VhJdpowerLogo.vue', () => {
  let wrapper;

  it('renders default VhJdpowerLogo', () => {
    wrapper = shallowMount(VhJdpowerLogo);

    expect(wrapper).toMatchSnapshot();
  });
});
