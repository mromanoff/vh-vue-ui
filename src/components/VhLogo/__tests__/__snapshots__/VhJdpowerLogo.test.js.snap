// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VhJdpowerLogo.vue renders default VhJdpowerLogo 1`] = `<div><img src="https://www.vehiclehistory.com/images/jd-power.png" alt="J.D. Power Logo" class="JdpowerLogo"></div>`;
